StandaloneMmPkg/Library/StandaloneMmHobLib/StandaloneMmHobLib.c

```diff
@@ -191,6 +191,10 @@ GetNextGuidHob (
     GuidHob.Raw = GET_NEXT_HOB (GuidHob);
   }
 
+  if ((GuidHob.Raw == NULL) && (HobStart == GetHobList ())) {
+    DEBUG ((DEBUG_WARN, "MM: HOB not found for GUID: %g\n", Guid));

```

Comment:
MM: GUIDed HOB [%g] not found in the entire HOB list.

Reply:
updated